learntools/sql/ex5.py

```diff
@@ -135,7 +135,7 @@ def check(self, results):
 
 # Set up the query (cancel the query if it would use too much of 
 # your quota, with the limit set to 1 GB)
-safe_config = bigquery.QueryJobConfig(maximum_bytes_billed=1e9)
+safe_config = bigquery.QueryJobConfig(maximum_bytes_billed=10**9)
```

Comment:
Geez. I somefixed other occurrences of 1e9 in some of these files and missed others.

Thanks for catching it.